### PR TITLE
ironscales - add defaults

### PR DESCRIPTION
**Pull request by melkabetz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Ironscales/Integrations/Ironscales/Ironscales.yml [hunk 1]
```diff
@@ -45,8 +45,10 @@ configuration:
   required: false
   type: 13
 description: IRONSCLAES, a self-learning email security platform integration
+defaultvalue: Ironscales
```
Comment (barchen1) [inline Packs/Ironscales/Integrations/Ironscales/Ironscales.yml:48]: Why did you add `defaultvalue`?
You can remove it

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please create RN file for your changes, you can take a look here: https://xsoar.pan.dev/docs/documentation/release-notes

Let me know if you need help

File: Packs/Ironscales/Scripts/IronscalesEmailFieldTrigger.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+commonfields:
+  id: IronscalesEmailFieldTrigger
+  version: 26
```
Comment (barchen1) [inline Packs/Ironscales/Scripts/IronscalesEmailFieldTrigger.yml:3]:
```suggestion
  version: -1
```

**Review by barchen1 — CHANGES_REQUESTED** [review]
1. Please fix the validation issues here: https://app.circleci.com/pipelines/github/demisto/content/116873/workflows/5b3d0c5d-c8c2-4485-b826-64fbda73cd05/jobs/428071

2. The script should be in this path(need to add a folder with the script name under Scripts folder)
`Packs/Ironscales/Scripts/IronscalesEmailFieldTrigger/IronscalesEmailFieldTrigger.yml`
3. Please add a readme file for the script and unitest file (`README.md`, `IronscalesEmailFieldTrigger_test.py`)
4. Please add a new release-notes file and update the `pack_metadata` file accordingly.

Let me know if you need help

File: Packs/Ironscales/Integrations/Ironscales/Ironscales.yml [hunk 3]
```diff
@@ -21,6 +21,7 @@ configuration:
   required: true
   type: 0
 - defaultvalue: '50'
+  display: Maximum number of incidents to fetch every time
```
Comment (barchen1) [inline Packs/Ironscales/Integrations/Ironscales/Ironscales.yml:24]:
```suggestion
  display: Maximum number of incidents per fetch
```

File: Packs/Ironscales/Scripts/IronscalesEmailFieldTrigger/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (barchen1) [inline Packs/Ironscales/Scripts/IronscalesEmailFieldTrigger/README.md:1]: Please add some short data about the script.

File: Packs/Ironscales/ReleaseNotes/1_1_0.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+#### Incident Fields
+- **Ironscales-Choose-Classification**
+
+#### Incident Types
+- **Ironscales**
+
+#### Integrations
+##### Ironscales
+- %%UPDATE_RN%%
```
Comment (barchen1) [inline Packs/Ironscales/ReleaseNotes/1_1_0.md:10]: Please add a note about the changes you made in the yml file.

**Review by barchen1 — CHANGES_REQUESTED** [review]

File: Packs/Ironscales/ReleaseNotes/1_1_0.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+#### Integrations
+##### Ironscales
+- Fixed a typo in the context output of ***ironscales-classify-incident*** command.
+- Added a default mapper to the integration.
+- Updated the Docker image to: *demisto/python3:3.9.7.24076*.
+
+#### Scripts
+##### New: IronscalesEmailFieldTrigger
+- Added new automation - Automatically changes email field when choosing classification.
+
```
Comment (bakatzir) [inline Packs/Ironscales/ReleaseNotes/1_1_0.md:17]:
```suggestion

#### Incident Fields
- **Ironscales-Choose-Classification**

#### Incident Types
- **Ironscales**

#### Integrations
##### Ironscales
- Fixed a typo in the context output of the ***ironscales-classify-incident*** command.
- Added a default mapper to the integration.
- Updated the Docker image to: *demisto/python3:3.9.7.24076*.

#### Scripts
##### New: IronscalesEmailFieldTrigger
- Automatically changes email field when choosing a classification.

```

**Review by barchen1 — APPROVED** [review]

**Review by bakatzir — APPROVED** [review]
failing on the change of the context path that was misspelled. merging.
```
Packs/Ironscales/Integrations/Ironscales/Ironscales.yml: [BC102] - Possible backwards compatibility break, You've changed the context in the file,please undo. the command is:
ironscales-classify-incident
```

**Pull request merged**